daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/infoset/Infoset.scala

```diff
@@ -248,8 +251,14 @@ class XMLTextInfosetOutputter private (outputter: SXMLTextInfosetOutputter)
    * @param pretty enable or disable pretty printing. Pretty printing will only
    *               insert indentation and newlines where it will not affect the
    *               content of the XML.
+   * @param xmlOutputStyle determine whether to wrap values of elements of type
+   *                       xs:string in CDATA chunks in order to preserve
+   *                       whitespace.
    */
-  def this(os: java.io.OutputStream, pretty: Boolean) = this(new SXMLTextInfosetOutputter(os, pretty))
+  def this(os: java.io.OutputStream, pretty: Boolean,
+    xmlOutputStyle: XMLOutputStyle.Value = XMLOutputStyle.Standard) = {
+    this(new SXMLTextInfosetOutputter(os, pretty, XMLOutputStyleConversions.modeToScala(xmlOutputStyle)))
```

Comment:
We aren't doing the null usage check here. If we are going to do that, we should be consistent.

Reply:
The enum conversion function will detect null as well although Scala code would have to do something unusual to pass null anyway.